lib/code_builder.dart

```diff
@@ -44,60 +44,47 @@ part 'src/builders/method_builder.dart';
 part 'src/builders/parameter_builder.dart';
 part 'src/builders/statement_builder.dart';
 part 'src/builders/type_builder.dart';
-
 part 'src/pretty_printer.dart';
 part 'src/scope.dart';
 
+final DartFormatter _dartfmt = new DartFormatter();
+
+// Simplifies some of the builders by having a mutable node we clone from.
+/// Returns [source] formatted by `dartfmt`.
+@visibleForTesting
+String dartfmt(String source) => _dartfmt.format(source);
+
+// Creates a defensive copy of an AST node.
```

Comment:
Is cloning only useful for defensive purposes? If not, I'd drop "defensive". Also, if the cloning is deep, I'd note that.